core/templates/pages/technical-feedback-dashboard-page/technical-feedback-dashboard-page.component.html

```
@@ -0,0 +1,28 @@
+<div>
+  <!-- TODO(#24716) Create UI components for
+   Technical Feedback Dashboard and wire up with the
+   backend 
+   Once the UI components are created, remove this comment and the below code.-->
+  {{ 'I18N_TOPNAV_TECHNICAL_FEEDBACK_DASHBOARD' | translate }}
+  <div class="button-container">
+    <button
+      class="btn btn-primary"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)